packages/integrations/tailwind/package.json

```diff
@@ -29,6 +29,7 @@
   },
   "devDependencies": {
     "@types/tailwindcss": "^3.0.9",
+    "@proload/core": "^0.2.2",
```

Comment:
This is in `devDependencies`, but looks like it should be in `dependencies`.

I also wonder if it's worth using the config loader from `tailwindcss` directly? [See `tailwindcss/resolveConfig.js`](https://github.com/tailwindlabs/tailwindcss/blob/master/resolveConfig.js). That way if they add support for something `@proload/core` doesn't have it will at least stay in sync.

Reply:
This doesn't do what you might hope! `resolveConfig` is _actually_ used to log all Tailwind core configurations for use in your JS. It does _not_ attempt to resolve any `tailwind.config.*` files in your project (just verified on my machine to confirm). Nuxt's tailwind plugin has to do some similarly custom logic to load tailwind config files I'm afraid.

Reply:
Ah okay, wishful thinking! Bummer. I'm definitely pleased with `@proload/core`, then!